src/plugins/memdump/memdump.cpp

```diff
@@ -277,6 +278,12 @@ bool dump_memory_region(
     fclose(fp);
 
     chk_str = g_checksum_get_string(checksum);
+    dump_hash.assign(chk_str);
```

Comment:
Declare `dump_hash` at the beginning and `assign` here as we have `goto` statements above.